libcipher/caesar.py

```diff
@@ -0,0 +1,64 @@
+__author__ = 'kskrishnasangeeth'
+
+import random
+import sys
+
+
+def encrypt(message, key=None):
+    """Returns the encrypted messsage for the string passed as argument.
+
+    Uses a random key for generating the cipher text
+
+    Args:
+        message (str) : The message to be encrypted.
+        key (int) : The key based on which encryption is performed. Is optional value.
+
+    Return:
+        str: the encrypted message """
+
+    if key is None:
+        key = random.randint(1, 25)
+
+    if key <= 0 or key >= 27:
+        print('Key should be having value between 1 and 26 ')
+        sys.exit(0)
```

Comment:
You should never quit from a library function. Raise an exception to the caller, and let the caller decide what to do.